src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -0,0 +1,67 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=method-hidden
+#pylint: disable=line-too-long
+#pylint: disable=bad-continuation
```

Comment:
do not disable `line-too-long` and `bad-continuation` unless you have very good reasons. they're easy to fix and important to readability.